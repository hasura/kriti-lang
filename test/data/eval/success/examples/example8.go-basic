[
  {{ range _, $var := $ }}
   {{ $var.eyeColor }}
  {{ end }}
]
