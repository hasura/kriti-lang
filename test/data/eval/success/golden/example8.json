[
    [
        "blue",
        "brown",
        "blue",
        "brown",
        "blue"
    ]
]